### ISSUE TITLE
ExplicitJobPermissions false negative: reusable workflow call

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#128]
### Version

0.3.1

### Reproduction steps

1. File `.github/workflows/x.yml` with the following content:

```yaml
permissions:
  contents: read
  checks: write
  statuses: write
  security-events: write
  actions: read

jobs:

  validate:
    name: "🦺 Validation"
    uses: TWiStErRob/github-workflows/.github/workflows/validate.yml@94f2ec4f96e146200f801fe7e09310c79cb2d2b3 # v2

  build:
    name: "🔨 Build & Verify"
    needs: validate
    uses: ./.github/workflows/CI-build.yml

  instrumentation:
    name: "🧪 Instrumentation Tests on" # / API ${{ matrix.api }} will be appended by used workflow.
    needs: validate
    uses: ./.github/workflows/CI-instrumentation.yml

```

2. Run `ghlint` with the following arguments:
   ```shell
   ghlint --console .github/workflows/x.yml
   ```

### Expected behavior

ExplicitJobPermissions

### Actual behavior

\-
